src/app/components/BibPage/BibDetails.jsx

```diff
@@ -319,6 +339,57 @@ class BibDetails extends React.Component {
         }
       }
 
+      // Note field rendering as array of objects instead of an array of strings.
+      // Parse the original and new note format.
+      // Original format: ['string1', 'string2']
+      // 2018 format:
+      //    [{'noteType': 'string',
+      //     'prefLabel': 'string',
+      //     '@type': 'bf:Note'},
+      //    {...}]
+      if (fieldLabel === 'Contents') {
+        const note = this.getNote(this.props.bib);
+        let notes;
+        if (note) {
+          if (note.length === 1) {
```

Comment:
Could there be the case that note[0] is an object, but here it passes because there could be the situation where there's only one object in the array? So this would pass.